doc/source/groupby.rst

```diff
@@ -620,6 +628,14 @@ We can also visually compare the original and transformed data sets.
    @savefig groupby_transform_plot.png
    compare.plot()
 
+Transformation functions that have lower dimension outputs are broadcast to
+match the shape of the input array.
+
+.. ipython:: python
+
+   extremum = lambda x: x.max() - x.min()
+   transformed = ts.groupby(key).transform(extremum)
```

Comment:
Can you show here the output of `transformed`? (or just remove the `transformed =` if this variable is not further used).

It would be nice to also show that you can use `transform(np.min)` or `tranform('min')`.